packages/migrate/test/scenarios/0.37/main.cadl

```diff
@@ -0,0 +1,95 @@
+import "@cadl-lang/rest";
```

Comment:
It would be nice to have a configuration file to migrate

Reply:
You mean cadl-project.yaml? There is one. I have to add content migration for that one.

Reply:
Agreed. It would be very good to be config data-driven rather than code-driven. Will track as an improvement for migration tool